### DISCUSSION TITLE
[FeatureRequest] Brush/zooming navigation for time series

### DISCUSSION THREAD

**Discussion by LeeZChuan** [grafana/grafana#38265] [Dashboards, Panels & Visualizations]
**Adding a new interactive way of dragging a box to the time series line chart**:

Now chart:
![image](https://user-images.githubusercontent.com/48050314/116360820-e3496000-a832-11eb-8e86-d9c6dc3e8f93.png)

Demand chart：I want add the drag box under the Now chart ；
![image](https://user-images.githubusercontent.com/48050314/116360955-0aa02d00-a833-11eb-8e76-5769d734242e.png)

The sample graph is come from https://echarts.apache.org/examples/zh/editor.html?c=line-aqi

**Why is this needed**:

Because it can make user easily contor the line chart. And it can make the user can not lose the data message to view the line chart.

By the way,this issue is your engineer suggested me to come forward.
[https://community.grafana.com/t/how-to-add-the-drag-box-in-grafana-chart/46651](url)

THANKS!

**Comment by leeoniya** [comment]
the vis mechanics for this are already coded for the core component: https://leeoniya.github.io/uPlot/demos/zoom-ranger-grips.html

**Comment by LeeZChuan** [comment]
I HOPE IT COME SOON. THANKS~

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
